src/tasks/tests-task.ts

```diff
@@ -0,0 +1,21 @@
+import { ITask, IProject, ICheckupResult } from '../interfaces';
+import FileSearcherTask from '../file-searcher-task';
+import * as path from 'path';
+
+export default class TestsTask extends FileSearcherTask implements ITask {
+  constructor(project: IProject, result: ICheckupResult) {
+    const SEARCH_PATTERNS = {
+      unit: [path.join(project.root, '**/tests/unit/**/*.js')],
```

Comment:
I'd prefer to not do this over just configuring the searcher itself with the right base directory in which to search. `globby` accepts an option `cwd` that is the root of the search. Can we not just do that instead?

Reply:
makes total sense!